fastlane/Fastfile

```diff
@@ -147,6 +147,14 @@ lane :verify_docs do
   end
 end
 
+desc "Generates the actions.md that will get used on https://docs.fastlane.tools"
+desc "This will NOT also automatically submit a pull request to fastlane/docs"
+desc "This SHOULD only be used for testing changes to the docs"
+lane :generate_docs do
```

Comment:
In accordance with the description above the lane, please rename this to `test_generate_docs`